homeassistant/components/solarlog/config_flow.py

```diff
@@ -86,21 +91,31 @@ async def async_step_user(self, user_input=None):
                     vol.Required(
                         CONF_HOST, default=user_input.get(CONF_HOST, DEFAULT_HOST)
                     ): str,
+                    vol.Required("extended_data", default=False): bool,
                 }
             ),
             errors=self._errors,
         )
 
     async def async_step_import(self, user_input=None):
```

Comment:
```suggestion
    async def async_step_import(self, user_input: dict[str, Any]):
```
If user_input would ever be None, the code would fail